components/Footer.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import Image from 'next/image';
+import { FooterLogo, Call, Mail, Location } from '../assets';
+import Link from 'next/link';
+import SocialLink from './SocialLink';
+import SocialMediaLinks from '../data/social';
+
+const Footer = () => {
+  return (
+    <footer className="bg-amcovad-tertiary bg-cover bg-left lg:bg-top  bg-[url('../assets/images/footer-hexagons.png')]">
+      <div className="px-4 pt-16 mx-auto sm:max-w-xl md:max-w-full pb-24 lg:max-w-screen-xl md:px-24 ">
+        <div className="container px-2 py-4 mx-auto">
+          <div className="grid gap-4 lg:gap-20 row-gap-10 mb-8 lg:grid-cols-6">
+            <div className="lg:col-span-2">
+              <Link passHref href="/">
+                <span className="inline-flex items-center cursor-pointer">
+                  <Image src={FooterLogo} alt="footer logo" />
+                </span>
+              </Link>
+              <div className="mt-4 lg:max-w-sm">
+                <div className="flex flex-col justify-between pt-1 pb-10 sm:flex-row">
+                  <div className="flex items-center mt-4 space-x-4 sm:mt-0">
+                    {SocialMediaLinks.map((data, index) => {
```

Comment:
In Javascript, variables should start with small letters.

```suggestion
                    {socialMediaLinks.map((data, index) => {
```